links/main.css

```diff
@@ -151,7 +151,7 @@ body #operator #icon:hover { opacity: 0.5 }
 
 body.guest #operator #wrapper { display:none; }
 
-html.night { filter: invert(100%); background: #111; }
-html.night body #feed .entry img.media { filter: invert(100%); background: #666; }
-html.night body #feed .entry video.media { filter: invert(100%); }
+html.night { filter: hue-rotate(180deg) invert(); background: #111; }
+html.night body #feed .entry img.media { filter: invert(); background: #666; }
+html.night body #feed .entry video.media { filter: invert(); }
```

Comment:
The hue of media needs to be shifted again, otherwise this happens: 

![image](https://user-images.githubusercontent.com/1200380/32945134-8a5f0db2-cb92-11e7-833b-c1de398bb366.png)

Reply:
aaaaa